src/native/corehost/version.h

```diff
@@ -15,12 +15,6 @@ struct version_t
     int get_major() const { return m_major; }
     int get_minor() const { return m_minor; }
     int get_build() const { return m_build; }
-    int get_revision() const { return m_revision; }
```

Comment:
It looks odd for `get_revision` getter to be missing here. I think it would look better to keep it, but comment it out.